### PR TITLE
add static var support for neo.compiler.msil

### PR DESCRIPTION
**Pull request by lightszero**
add static var support for neo.compiler.msil

### CODE REVIEW

File: Neo.Compiler.MSIL/MSIL/Converter.cs [hunk 1]
```diff
@@ -1016,17 +1035,43 @@ private int ConvertCode(ILMethod method, OpCode src, NeoMethod to)
...
+
+                            _Insert1(VM.OpCode.PICKITEM,"",to);
+
+                            //throw new Exception("Just allow defined a static variable with readonly." + d.FullName);
                         }
                     }
                     break;
+                case CodeEx.Stsfld:
+                    {
+                        _Convert1by1(VM.OpCode.NOP, src, to);
```
Comment (erikzhang) [inline Neo.Compiler.MSIL/MSIL/Converter.cs]: Why `NOP`?

Comment (lightszero) [reply Neo.Compiler.MSIL/MSIL/Converter.cs]: insert a nop pos for quick find avm when i debug it.will remove later

Comment (shargon) [reply Neo.Compiler.MSIL/MSIL/Converter.cs]: Maybe we can detect if is debug or release?

Comment (lightszero) [reply Neo.Compiler.MSIL/MSIL/Converter.cs]: @shargon that idea is good.could be useful

Comment (erikzhang) [reply Neo.Compiler.MSIL/MSIL/Converter.cs]: #if DEBUG
    _Convert1by1(VM.OpCode.NOP, src, to);
#endif

**Pull request merged**